src/form-layout-grid.scss

```diff
@@ -0,0 +1,61 @@
+@import "./mixins";
+@import "./layout-grid.scss";
+
+$block: #{$fd-namespace}-form-layout-grid;
+$blockCol: #{$fd-namespace}-col;
+$blockContainer: #{$fd-namespace}-container;
```

Comment:
maybe it is worth bringing the file `_container.scss` and merge it with this file?